tests/jq_fuzz_execute.cpp

```diff
@@ -28,8 +28,11 @@ extern "C" int LLVMFuzzerTestOneInput(uint8_t *data, size_t size) {
         }
         jv_free(next);
       }
-
-      // Do not free "input" as this is handled by jq_teardown.
+      else {
+        // Only free if input as invalid as otherwise jq_teardown
+        // frees it.
```

Comment:
So if understand it correctly: `jq_parse` will always return something allocated. If valid the input argument will end up on the stack by `jq_start` and freed during teardown. If invalid it needs to be freed explicitly?

Reply:
Yes, that's how I understand it.

Reply:
"is invalid"?